EmployeeList/Views/EmployeeListView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  EmployeeListView.swift
+//  EmployeeList
+//
+//  Created by Wang Siman on 1/29/21.
+//  Copyright © 2021 None. All rights reserved.
+//
+
+import SwiftUI
+
+struct EmployeeListView: View {
+    @ObservedObject private(set) var employeeListViewModel: EmployeeListViewModel
+    
+    var body: some View {
+        NavigationView {
+            List {
+                ForEach(employeeListViewModel.sortedTeams, id: \.name) { team in
```

Comment:
Why isn't the Team Identifiable?

Reply:
There are actually two employees whose team names are "Point of Sale" and "Point of Sale Platform". They are seen as two different teams while they probably be the same. I expect I can fetch team list from Square that specifies team IDs.